.circleci/config.yml

```
@@ -1,576 +1,20 @@
-      - win_e2e_tests:
-          matrix:
-            parameters:
-              python_version: ["3.6", "3.7", "3.8"]
-      - unit_tests:
-          matrix:
-            parameters:
-              python_version: ["3.6", "3.7", "3.8"]
-      - win_unit_tests:
-          matrix:
-            parameters:
-              python_version: ["3.6", "3.7", "3.8"]
-      - lint:
-          matrix:
-            parameters:
-              python_version: ["3.6", "3.7", "3.8"]
-      - pip_compile:
-          matrix:
-            parameters:
-              python_version: ["3.6", "3.7", "3.8"]
-      - win_pip_compile:
-          matrix:
-            parameters:
-              python_version: ["3.6", "3.7", "3.8"]
-      - build_docs
-      - docs_linkcheck
-      - all_circleci_checks_succeeded:
-          requires:
-            - e2e_tests
-            # win_e2e_tests-3.6 is not required due to the error
-            # `Cargo, the Rust package manager, is not installed or is not on PATH`.
-            # - win_e2e_tests
-            - win_e2e_tests-3.7
-            - win_e2e_tests-3.8
-            - unit_tests
-            - win_unit_tests
-            - lint
-            - pip_compile
-            - win_pip_compile
-            - build_docs
-            - docs_linkcheck
-
-  main_updated:
-    unless: <<pipeline.parameters.release_kedro>>  # don't run if 'release_kedro' flag is set
-    jobs:
-      - sync:
-          filters:
-            branches:
-              only: main
-      - viz_build:
-          filters:
-            branches:
-              only: main
-
-  hourly_pr_merge:
-    triggers:
-      - schedule:
-          cron: 0 * * * *
-          filters:
-            branches:
-              only: main
-    jobs:
-      - merge_pr_to_develop
-
-  # Python versions that are supported on `main`.
-  nightly_build_main:
-    triggers:
-      - schedule:
-          cron: 30 2 * * *
-          filters:
-            branches:
-              only: main
-    jobs:
-      - build_docker_image:
-          matrix:
-            parameters:
-              python_version: ["3.6", "3.7", "3.8"]
-      - build_kedro:
-          matrix:
-            parameters:
-              python_version: ["3.6", "3.7", "3.8"]
-          requires:
-            - build_docker_image-<<matrix.python_version>>
-
-  # Python version that are *only* supported on `develop`.
-  nightly_build_develop:
-    triggers:
-      - schedule:
-          cron: 30 2 * * *
-          filters:
-            branches:
-              only: develop
-    jobs:
-      - build_docker_image:
-          matrix:
-            parameters:
-              python_version: ["3.9"]
-      - build_kedro:
-          matrix:
-            parameters:
-              python_version: ["3.9"]
-          requires:
-            - build_docker_image-<<matrix.python_version>>
-
-  kedro_release:
-    when: <<pipeline.parameters.release_kedro>>  # only run if 'release_kedro' flag is set
+  always-run:
```

```suggestion
  always_run:
```
just to match the other workflows